RandBLAS/trig_skops.hh

```diff
@@ -125,29 +103,29 @@ namespace RandBLAS {
         //TODO: discuss precise semantics of `selected_rows` in this function
         if(layout == blas::Layout::ColMajor) {
             for (int64_t i=0; i < d; i++) {
-                randblas_error_if_msg(selected_rows[i] == top,
-                                      "The list of provided indices should be unique");
                 if (selected_rows[i] != top) {
                     // Use BLAS swap to swap the entire rows
                     // Swapping row 'selected' with row 'top'
                     blas::swap(cols, &A[top], rows, &A[selected_rows[i]], rows);
                 }
+                else
+                    continue;
```

Comment:
Minor point. Since the "else continue" isn't doing anything functional, please use the following style instead.
```
                if (selected_rows[i] != top) {
                    // Use BLAS swap to swap the entire rows
                    // Swapping row 'selected' with row 'top'
                    blas::swap(cols, &A[top], rows, &A[selected_rows[i]], rows);
                } // else, continue.
```